src/discord-cluster-manager/report.py

```diff
@@ -43,31 +46,51 @@ async def _send_split_log(thread: discord.Thread, partial_message: str, header:
         return ""
 
 
-async def _generate_compile_report(thread: discord.Thread, comp: CompileResult):
+@dataclasses.dataclass
+class Text:
+    text: str
+
+
+@dataclasses.dataclass
+class Log:
```

Comment:
maybe just a quick comment on when someone should instantiate Text vs Log

Reply:
done